prac_05/word_occurences.py

```diff
@@ -0,0 +1,21 @@
+word_occurences = {}
+
+text = str(input("Text: "))
+words = text.split()
+
+for word in words:
+    if word in word_occurences:
+        word_occurences[word] += 1
+    else:
+        word_occurences[word] = 1
+
+word_occurences_keys = [key for key in word_occurences]
+word_occurences_keys.sort()
```

Comment:
Interesting, but there is a simpler way to sort the words, by moving the line to the top after you split the text, before you do the printing of the sentence.